lib/gtfs/trip.ex

```diff
@@ -20,11 +21,14 @@ defmodule Gtfs.Trip do
     :route_id
   ]
 
-  @spec from_csv_row(%{required(String.t()) => String.t()}) :: t()
-  def from_csv_row(row) do
+  @spec from_csv_row(Csv.row()) :: t()
+  def from_csv_row(trip_row) do
     %__MODULE__{
-      id: row["trip_id"],
-      route_id: row["route_id"]
+      id: trip_row["trip_id"],
+      route_id: trip_row["route_id"]
     }
   end
+
+  @spec in_id_set?(Csv.row(), MapSet.t(id())) :: boolean
+  def in_id_set?(trip_row, id_set), do: MapSet.member?(id_set, trip_row["route_id"])
```

Comment:
This is checking whether the route is in a set of routes, but the type and names say it's for a set of trips.

I know Paul just suggested these names, but I think the `member_of_routes?(row, route_ids)` name was clearer. (This argument order is better, though.)

(Ditto for `route_pattern.ex`)

Reply:
It's not a MapSet of Routes, but of route IDs. `in_route_id_set?` maybe?

Reply:
I like that idea.